avocado/utils/lv_utils.py

```diff
@@ -125,7 +125,7 @@ def vg_ramdisk(disk, vg_name, ramdisk_vg_size,
             process.run("losetup %s %s" %
                         (loop_device, ramdisk_filename), sudo=True)
         LOGGER.debug("Creating physical volume %s", loop_device)
-        process.run("pvcreate %s" % loop_device, sudo=True)
+        process.run("pvcreate -y %s" % loop_device, sudo=True)
```

Comment:
OK I guess this is when using an existing `disk` argument, right? Never mind, it should not break anything so let's do this.